chef/data_bags/crowbar/migrate/cinder/033_add_max_min_size_timeout_pool.rb

```diff
@@ -0,0 +1,19 @@
+def upgrade ta, td, a, d
+  if a['max_pool_size'].nil?
+    a['max_pool_size'] = ta['max_pool_size']
+  end
+  if a['max_overflow'].nil?
+    a['max_overflow'] = ta['max_overflow']
+  end
+  if a['pool_timeout'].nil?
+    a['pool_timeout'] = ta['pool_timeout']
+  end
+  return a, d
+end
+
+def downgrade ta, td, a, d
+  a.delete 'max_pool_size'
```

Comment:
this needs to be 

unless ta.has_key? 'max_pool_size'

around (to have the equivalent behavior of the upgrade function)